### PR TITLE
Add permission error section to FAQ

### PR DESCRIPTION
**Pull request by gliptak**
https://github.com/Microsoft/WSL/issues/3181

### CODE REVIEW

File: WSL/troubleshooting.md [hunk 1]
```diff
@@ -247,3 +247,19 @@ This error is related to being in a bad install state. Please complete the follo
 - Update your version of Windows by going to Settings, Updates, and clicking 'Check for Updates'
 
 - If both of those fail and you need to access WSL please consider upgrading in place by reinstalling Windows 10 using installation media and selecting 'Keep Everything' to ensure your apps and files are preserved. You can find instructions on how to do so at the [Reinstall Windows 10 page](https://support.microsoft.com/help/4000735/windows-10-reinstall).
+
+### Correct (SSH related) permission errors
+
+Like
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md]: I'd recommend changing the wording here to:

`If you're seeing this error:`

File: WSL/troubleshooting.md [hunk 2]
```diff
@@ -247,3 +247,19 @@ This error is related to being in a bad install state. Please complete the follo
...
+```
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+Permissions 0777 for '/home/artur/.ssh/private-key.pem' are too open.
+```
+To correct appent to ```/etc/wsl.conf``` file
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md]: I'd recommend changing the wording here to: 

`To fix this, append the following to the the ```/etc/wsl.conf``` file:`

File: WSL/troubleshooting.md [hunk 3]
```diff
@@ -247,3 +247,19 @@ This error is related to being in a bad install state. Please complete the follo
...
+```
+[automount]
+enabled = true
+options = metadata,uid=1000,gid=1000,umask=0022
+```
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md]: I'd also recommend adding: 

`Please note that adding this command will include metadata and modify the file permissions on the Windows files seen from WSL. Please see the [File System Permissions](./file-permissions.md) for more information.`

File: WSL/troubleshooting.md (hunk 1 above)
Comment (gliptak) [reply WSL/troubleshooting.md]: updated

**Comment by gliptak** [comment]
@craigloewen-msft thank you for the new verbiage

**Comment by craigloewen-msft** [comment]
Done! Thank you for the awesome content! :)

**Pull request merged**